src/enums/roles.ts

```diff
@@ -1,6 +1,7 @@
 export enum Roles {
-  COMPLETING_PARTY = 'Completing Party',
-  INCORPORATOR = 'Incorporator',
-  DIRECTOR = 'Director',
-  SUBSCRIBER = 'Subscriber',
+  COMPLETING_PARTY = 'completing party',
+  INCORPORATOR = 'incorporator',
+  DIRECTOR = 'director',
+  SUBSCRIBER = 'subscriber',
+  CUSTODIAN = 'custodian'
```

Comment:
As commented above, this is to keep in alignment with the data in our database vs filing json values.